sfdc/src/test/java/com/jalasoft/sfdc/steps/PriceBook.java

```diff
@@ -0,0 +1,40 @@
+package com.jalasoft.sfdc.steps;
+
+import com.jalasoft.sfdc.ui.PageFactory;
+import com.jalasoft.sfdc.ui.pages.AppLauncher;
+import com.jalasoft.sfdc.ui.pages.DetailPage;
+import com.jalasoft.sfdc.ui.pages.ListHomePage;
+import com.jalasoft.sfdc.ui.pages.home.HomePage;
+import com.jalasoft.sfdc.ui.pages.priceBook.PriceBookForm;
+import com.jalasoft.sfdc.ui.pages.priceBook.PriceBookFormLight;
+import cucumber.api.PendingException;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.When;
+
+import java.util.List;
+
+public class PriceBook {
+    HomePage homePage;
+    AppLauncher appLauncher;
+    ListHomePage listHomePage;
+
+
+    @Given("^I go to PriceBook Home Page$")
+    public void iGoToPriceBookHomePage() throws Throwable {
+        homePage = PageFactory.getHomePage();
+        appLauncher=homePage.topMenu.gotToAppLaucher();
+        listHomePage=appLauncher.gotToBookPage();
+    }
+
+    @And("^I click on New Button$")
+    public void iClickOnNewButton() throws Throwable {
+        listHomePage.clickSaveNewButton();
```

Comment:
this should return PriceBookFormPage

Reply:
Ah I see you do not have ListHome pages for features, there is only 1, that is why you are not returning any page.